klayout/drc/rule_decks/tail.drc

```diff
@@ -0,0 +1,23 @@
+################################################################################################
```

Comment:
is that tested? (if not should we remove it?)

Reply:
ah my bad this is just for logging something at the end of all deck?

Reply:
any reason we don't do this logging in main itself?

Reply:
are `main` and `tail` existing klayout convention? or something that only applies to this project?

Reply:
No, I came up with those after I had a discussion with @mithro to try to split the rule deck to parts to make it easier to understand and debug.

Reply:
Make sense, thanks for the detailed explanation.